webrtc.html

```diff
@@ -8352,17 +8352,25 @@ <h3 id="sec.identity-proxy-assertion-request">Requesting Identity
           provided.</p>
         </li>
         <li>
-          <p>The IdP returns a Promise to the <code>RTCPeerConnection</code>.
-          If the user has been authenticated by the IdP, and the IdP is willing
-          to generate an identity assertion, the IdP resolves the promise with
-          an identity assertion in the form of an
+          <p>The IdP proxy returns a Promise to the
+          <code>RTCPeerConnection</code> and generates an identity assertion
+          asynchronously.</p>
+          <p>If the user has been authenticated by the IdP, and the IdP is
+          willing to generate an identity assertion, the IdP resolves the
```

Comment:
This part is a bit confusing. First it says ```IdP proxy ... generates an identity assertion``` and then ```if ... the IdP is willing to generate an identity assertion```. It looks like the IdP proxy has already generated that one.